google/__init__.py

```diff
@@ -28,4 +28,10 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 # pylint: disable=missing-docstring
-__import__('pkg_resources').declare_namespace(__name__)  # pragma: no cover
+from __future__ import absolute_import
```

Comment:
I actually pulled this from `google-auth-client` while I was trying to debug an issue with running tox in Python 3.5. This ended up not being the problem, but I am fairly sure that this is a slightly better approach (for the extremely rare case where someone does not have setuptools).